File: Java IMDB/src/main/java/imdb/entity/Film.java
package imdb.entity;

import javax.persistence.*;

@Entity
@Table(name = "films")
public class Film {
	private Integer id;
	private String Name;
	private String Director;
	private String Genre;
	private Integer Year;

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
	public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    @Column(nullable = false)
    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    @Column(nullable = false)
    public String getDirector() {
        return Director;
    }

    public void setDirector(String director) {
        Director = director;
    }

    @Column(nullable = false)
    public String getGenre() {
        return Genre;
    }

    public void setGenre(String genre) {
        Genre = genre;
    }

    @Column(nullable = false)
    public Integer getYear() {
        return Year;
    }

    public void setYear(Integer year) {
        Year = year;
    }

    public Film(String name, String director, String genre, Integer year) {
        this.Name = name;
        this.Director = director;
        this.Genre = genre;
        this.Year = year;
    }

    public Film() {

    }
}

File: PHP IMDB/src/AppBundle/Entity/Film.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Film
 *
 * @ORM\Table(name="films")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\FilmRepository")
 */
class Film
{
    /**
     * @var int
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     * @ORM\Column(type = "integer")
     */
    private $id;

    /**
     * @var string
     * @ORM\Column(name = "name", type="string", length=255, nullable=false)
     *
     */
    private $name;

    /**
     *
     * @var string
     * @ORM\Column(name="genre", type="string", length=255, nullable=false)
     *
     */
    private $genre;

    /**
     * @var string
     * @ORM\Column(name="director", type="string", length=255, nullable=false)
     *
     */
    private $director;

    /**
     * @var int
     * @ORM\Column(name="year")
     */
    private $year;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }


    /**
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * @param string $name
     * @return Film;
     */
    public function setName($name)
    {
        $this->name = $name;
        return $this;
    }

    /**
     * @return string
     */
    public function getGenre()
    {
        return $this->genre;
    }

    /**
     * @param string $genre
     * @return Film;
     */
    public function setGenre($genre)
    {
        $this->genre = $genre;
        return $this;
    }

    /**
     * @return string
     */
    public function getDirector()
    {
        return $this->director;
    }

    /**
     * @param string $director
     * @return Film;
     */
    public function setDirector($director)
    {
        $this->director = $director;
        return $this;
    }

    /**
     * @return mixed
     */
    public function getYear()
    {
        return $this->year;
    }

    /**
     * @param int $year
     * @return Film
     */
    public function setYear($year)
    {
        $this->year = $year;
        return $this;
    }


}

